pymilvus/client/prepare.py

```diff
@@ -625,10 +625,25 @@ def search_requests_with_expr(cls, collection_name, data, anns_field, param, lim
         param_copy = copy.deepcopy(param)
         metric_type = param_copy.pop("metric_type", "L2")
         params = param_copy.pop("params", {})
+        # tag edit
+        round_decimal = param_copy.pop("round_decimal",3)
+        print("test output",round_decimal)
+        if not isinstance(round_decimal,(int,str)):
```

Comment:
This judgment only needs to be executed once, and there is no need to judge once in each layer of functions.